stripe/src/main/java/com/stripe/android/PaymentSessionConfig.kt

```diff
@@ -75,8 +78,8 @@ data class PaymentSessionConfig internal constructor(
 
         /**
          * @param shippingMethodsRequired whether a [com.stripe.android.model.ShippingMethod]
```

Comment:
^ for `setShippingInfoRequired(shippingInfoRequired: Boolean): Builder` - Can we specify the default value is `true`?